libr/debug/p/native/linux/linux_debug.c

```diff
@@ -509,3 +514,1397 @@ RList *linux_desc_list (int pid) {
 	closedir (dd);
 	return ret;
 }
+
+
+/* Coredump  functions */
+
+static map_file_t mapping_file = {0,0};
+
+static char *prpsinfo_get_fname(FILE *f) {
+        char *p;
+        int c;
+        int pos;
+
+        p = malloc(16);
+        pos = 0;
+        while ((c = fgetc(f)) != EOF && c != '\0')
```

Comment:
why not an fread? If that can't work, can we at least put a limit on the maximum number of chars that can be read with this loop?